lighthouse/lighthouse.spec.js

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from '@playwright/test';
+import { playAudit } from 'playwright-lighthouse';
+import { chromium } from 'playwright';
+import { tmpdir } from 'os';
+
+test('Busola Lighthouse audit', async () => {
+  const context = await chromium.launchPersistentContext(tmpdir(), {
+    args: ['--remote-debugging-port=9222'],
+  });
+  const page = await context.newPage();
+
+  await page.goto('http://localhost:8080');
+
+  // on dev chromium may remember previous state - get back to clusters list
+  if (!page.url().endsWith('/clusters')) {
```

Comment:
it doesn't work on a first run. Address is just `localhost:8080`, and only after a while it gets redirected to `.../clusters`.

Reply:
Woah, I forgot that /clusters actually **doesn't** redirect you to current cluster... Fixed!

Reply:
Or actually... no, it does redirect to current cluster. I don't know why.